src/main/initNrfUtilProxyEnv.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import argv from './argv';
+
+export default () => {
+    const proxyServer = argv['proxy-server'] as string | undefined;
+    const httpProxy = proxyServer
+        ?.split(';')
+        .find(p => p.startsWith('http='))
+        ?.split('=')[1];
+
+    if (httpProxy) {
+        const proxy = `http://${argv['proxy-server']}`;
```

Comment:
With the current code this would have to be 
```suggestion
        const proxy = `http://${httpProxy]}`;
```

But as mentioned above I suggest that we only support `--proxy-server="foopy:8080"` and so this whole function can be simplified to 
```ts
export default () => {
    if (argv['proxy-server']) {
        const proxy = `http://${argv['proxy-server']}`;

   …
```